core/src/main/web/app/helpers/evaluatepluginmanager.js

```diff
@@ -129,9 +129,18 @@
                   .then(function(factory) {
                     if (factory !== undefined && factory.create !== undefined) {
                       return factory.create(evaluatorSettings).then(function(ev) { deferred.resolve(ev); });
-                    } else
-                      deferred.reject("no factory for evaluator plugin");
+                    } else {
+		      // or should someone pass in a scope to use, like with a method on the evaluatorLoadQueue?
+                      $rootScope.pluginId = name;
+		      var modal = $modal.open({backdrop: true,
+					       backdropClick: true,
+					       template: JST['helpers/plugin-load-error']()});
```

Comment:
You could also pass in a variable directly to the template invocation (especially considering it's not going to change at all after it's displayed), trying to find an example...